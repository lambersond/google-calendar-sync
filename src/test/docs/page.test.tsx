import { create } from 'react-test-renderer'
import Docs from '@/app/docs/page'

describe('app/layout', () => {
  it('should match layout snapshot', () => {
    const tree = create(<Docs />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
