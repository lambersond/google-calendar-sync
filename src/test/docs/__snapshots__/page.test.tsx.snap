// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/layout should match layout snapshot 1`] = `
<div
  className="px-10 py-8"
>
  <div
    className="flex flex-col w-full gap-6 items-center"
  >
    <p
      className="text-4xl text-slate-200"
    >
      Documents
    </p>
    <ul
      className="text-lg text-slate-100 flex flex-col items-center"
    >
      <li
        className="underline underline-offset-2"
      >
        <a
          href="/docs/privacy-policy"
        >
          Privacy Policy
        </a>
      </li>
      <li
        className="underline underline-offset-2"
      >
        <a
          href="/docs/terms-of-service"
        >
          Terms of Service
        </a>
      </li>
    </ul>
  </div>
</div>
`;
